codespeed/views.py

```diff
@@ -731,3 +739,239 @@ def add_json_results(request):
     logger.debug("add_json_results: completed")
 
     return HttpResponse("All result data saved successfully", status=202)
+
+
+def django_has_content_type():
+    return (django.VERSION[0] > 1 or
+            (django.VERSION[0] == 1 and django.VERSION[1] >= 6))
+
+
+def validate_results_request(data):
```

Comment:
Moved validate_results_request to validators.py but there is still another validation function that should follow the same template (raising a ValidationError on error instead of returning a message, bool tuple): def validate_result(item) from results.py. I could move that to validators.py and maybe propose a more generic way to do validation in another pull request.